JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js

```diff
@@ -0,0 +1,17 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+for (let i = 10; i <= 40; i++) {
```

Comment:
You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable